jcli/src/jcli_lib/transaction/simplified.rs

```diff
@@ -0,0 +1,68 @@
+use crate::jcli_lib::transaction::{common, Error};
+use crate::transaction;
+use crate::transaction::staging::Staging;
+use chain_impl_mockchain::transaction::Output;
+use jormungandr_lib::interfaces;
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+#[structopt(rename_all = "kebab-case")]
+pub struct SimplifiedTransaction {
+    /// the account to debit the funds from
+    #[structopt(name = "ACCOUNT")]
+    pub faucet_address: interfaces::Address,
```

Comment:
I don't really see anything specific to faucets here. Why not just `sender_account` or something?

Reply:
It was called `faucet_address` in the original script, so I kept it like that. But you are right, gonna rename it :)

Reply:
Don't pay too much attention to the original script naming. `jcli` is universal and does not care about the details of specific deployments.